src/modules/job-list/Makefile.am

```diff
@@ -9,7 +9,8 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir) \
 	-I$(top_srcdir)/src/include \
 	-I$(top_builddir)/src/common/libflux \
-	$(FLUX_SECURITY_CFLAGS) $(YAMLCPP_CFLAGS)
+	$(FLUX_SECURITY_CFLAGS) $(YAMLCPP_CFLAGS) \
```

Comment:
What's YAMLCPP_CFLAGS doing here?  Leftover from days gone by?  It's not defined.  Maybe drop that from here and other places it was cut and paste (job-archive, job-info, job-ingest, and kvs-watch).

Reply:
oh, haha!  I just assumed we were using it :P  i'll yoink it